app/TuistApp/Sources/Services/ServerURLService.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import Mockable
+import TuistSupport
+
+@Mockable
+protocol ServerURLServicing {
+    func serverURL() -> URL
+}
+
+struct ServerURLService: ServerURLServicing {
+    func serverURL() -> URL {
```

Comment:
Eventually, we should read the server url from the user settings, so it's possible to use the macOS app in on-premise environment as well

Reply:
Yup, that's correct